### PR TITLE
[sdks] Run Android tests on CI

### PR DESCRIPTION
**Pull request by luhenry**
It's assuming a device or emulator is already attached or running. On our CI, a new emulator is automatically started at the beginning of the job and shutdown at the end of it.

### CODE REVIEW

File: scripts/ci/run-jenkins.sh [hunk 1]
```diff
@@ -139,12 +139,34 @@ if [[ ${CI_TAGS} == *'product-sdks-android'* ]];
...
+        echo "DISABLE_WASM=1" >> sdks/Make.config
         ${TESTCMD} --label=provision-android --timeout=120m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-android
         if [[ ${CI_TAGS} == *'provision-mxe'* ]]; then
             ${TESTCMD} --label=provision-mxe --timeout=240m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-mxe
         fi
-        ${TESTCMD} --label=llvm --timeout=240m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-llvm-llvm{,win}{32,64}
+        ${TESTCMD} --label=provision-llvm --timeout=240m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-llvm-llvm{,win}{32,64}
         ${TESTCMD} --label=runtimes --timeout=120m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds package-android-{armeabi-v7a,arm64-v8a,x86,x86_64} package-android-host-{Darwin,mxe-Win64} package-android-cross-{arm,arm64,x86,x86_64}{,-win}
+        ${TESTCMD} --label=package --timeout=60m --fatal make -C sdks/android package
+        ${TESTCMD} --label=mini --timeout=60m make -C sdks/android check-mini
```
Comment (lewurm) [inline scripts/ci/run-jenkins.sh]: on what arch is it running? armeabi-v7a, arm64-v8a, x86 or x86_64? I guess it's whatever is available via adb?

what if we want to test 32bit and 64bit? it's possible to run an android app targeting armeabi-v7a on an arm64 device.

Comment (luhenry) [reply scripts/ci/run-jenkins.sh]: It's right now running on any architecture the emulator is. We should add it later, with the AOT configuration, AOT+LLVM configuration, etc.

**Comment by luhenry** [comment]
@monojenkins build failed

**Comment by luhenry** [comment]
@monojenkins build OS X i386

**Comment by luhenry** [comment]
@monojenkins build Windows i386

**Comment by luhenry** [comment]
@lambdageek I love the new OS X x64 C++ and Windows x64 C++!

**Comment by lambdageek** [comment]
@luhenry Thanks.  @jaykrell did all the difficult work. I just clicked some buttons in Jenkins.

**Pull request merged**